src/app/calculator/compressed-air/air-flow-conversion/air-flow-conversion.component.ts

```diff
@@ -102,21 +102,18 @@ export class AirFlowConversionComponent implements OnInit {
     return tmpCalculator;
   }
 
-  saveAssessmentCalculator(){
+  async saveAssessmentCalculator(){
```

Comment:
any reason saveAssessmentCalculator is still being used in this calculator when the service method is being used in the others?